commit 2638f0d81ad20df28af373dddfd5bb0c8db055b8
Author: victoria <victoriagbill@gmail.com>
Date:   Tue Oct 7 19:37:19 2014 -0500

    modified for patch

diff --git a/lab2_files/lab2_git/HelloWorld/HelloWorld.java b/lab2_files/lab2_git/HelloWorld/HelloWorld.java
index b316ea1..0a9b78c 100644
--- a/lab2_files/lab2_git/HelloWorld/HelloWorld.java
+++ b/lab2_files/lab2_git/HelloWorld/HelloWorld.java
@@ -1,7 +1,7 @@
 
 public class HelloWorld {
 	public static void main( String [] args){
-		System.out.println("Hello World!");
+		System.out.println("Hallo Welt!");
 	}
 
 }
